StellarAssetSdk.Tests/AssetIssuerTest.cs

```diff
@@ -35,4 +35,16 @@ public async Task Mint()
         Console.WriteLine($"{sgIssuer.AccountId}, {sgOperator.AccountId}, {alice.AccountId}");
         await Runner.Run(server, sgIssuer, sgOperator, alice);
     }
+
+    [TestMethod]
+    public async Task ClaimableBalance()
+    {
+        var server = StellarAssetSdk.Server.Local ();
+        var sgIssuer = await server.GenerateAndFund();
+        var sgOperator = await server.GenerateAndFund();
+        var alice = await server.GenerateAndFund();
+        Console.WriteLine($"{sgIssuer.AccountId}, {sgOperator.AccountId}, {alice.AccountId}");
+        await Runner.Run(server, sgIssuer, sgOperator, alice);
```

Comment:
This is again the same as the Mint test above. Instead you should check the account balances before and after the claiming claimable balances in `Runner`.